.github/workflows/squash_branches.yml

```diff
@@ -0,0 +1,79 @@
+name: Squash Branches
+on:
+    workflow_call:
+        inputs:
+            branch-prefix:
+                required: true
+                type: string
+            commit-message:
+                required: true
+                type: string
+        secrets:
+            GPG_SIGNING_KEY:
+                required: true
+            GH_TOKEN:
+                required: true
+            LASTMJS_GITHUB_TOKEN:
+                required: true
+
+jobs:
+    squash:
+        name: Squash Branches
+        runs-on: ubuntu-latest
+        env:
+            GPG_SIGNING_KEY: ${{ secrets.GPG_SIGNING_KEY }}
+            GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        steps:
+            - uses: actions/checkout@v4
+              with:
+                  ref: ${{ github.event.pull_request.head.ref || github.ref }}
```

Comment:
Remove all github.event.pull_request.head.ref if we aren't going to be using PRs to trigger workflows anymore

Reply:
done